test/gloss/test/core.clj

```diff
@@ -314,3 +314,8 @@
         dec)
       (string :utf-8 :suffix ","))
     "abc"))
+
+(deftest test-int24s
+  (test-roundtrip
+   [:int24 :int24-le :int24-be :uint24 :uint24-le :uint24-be]
+   [32779 32779 32779 32779 32779 32779]))
```

Comment:
Can you expand this a bit? Can we test negative numbers with the signed versions, and both BE and LE-ordered ByteBuffers?

Reply:
Just add bytebuffer output check.